conda_index/index/__init__.py

```diff
@@ -534,69 +534,74 @@ def index(
         ``index.html`` for each subdir.
         """
         if verbose:
-            level = logging.DEBUG
-        else:
-            level = logging.ERROR
-
-        logging_context = utils.LoggingContext(level, loggers=[__name__])
+            log.debug(
+                "ChannelIndex.index(verbose=...) is a no-op. Alter log levels for %s to control verbosity.",
+                __name__,
+            )
 
-        with logging_context:
```

Comment:
this section was dedented since our logging_context, inherited from conda-build, is a no-op